### PR TITLE
now image opacity working when images takes its time loading

### PR DESCRIPTION
**Pull request by FedericoChavazza**
## Links:

https://trello.com/c/XerAAPMu/13-loading-image

## What & Why:

now image has a lazy loading, providing better user experience, and more efficiency, with styles included

## Risks, Mitigation & Rollback:

- [x] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/components/viewComponents/Movie.js [hunk 1]
```diff
@@ -1,21 +1,28 @@
 import './styles.scss';
 import { getWatchlist } from 'utils/api';
+import { LazyLoadImage } from 'react-lazy-load-image-component';
+import 'react-lazy-load-image-component/src/effects/opacity.css';
+
+import { useState } from 'react';
 
 export function Movie({ title, date, img, icon, id, language }) {
   const watchlist = getWatchlist();
+  const [loaded, setLoaded] = useState(false);
```
Comment (JulianPasquale) [inline src/components/viewComponents/Movie.js]:
```suggestion
  const [imageLoaded, setImageLoaded] = useState(false);

```

File: src/components/viewComponents/Movie.js [hunk 2]
```diff
@@ -1,21 +1,28 @@
...
+      {!loaded ? (
+        <LazyLoadImage
+          alt=""
+          effect="opacity"
           src={`${process.env.REACT_APP_ORIGINAL_IMG}${img}`}
-          alt="movie"
         />
       ) : (
-        <div className="Movie-container--img-notfound"> </div>
+        <div className="Movie-container--img-loading" onLoad={() => setLoaded(true)}>
```
Comment (JulianPasquale) [inline src/components/viewComponents/Movie.js]: Why are you calling `setLoaded(true)` here? As far as I understand, this code will only run if `loaded` is true

Comment (FedericoChavazza) [reply src/components/viewComponents/Movie.js]: i have to delete this inmediatly

Comment (FedericoChavazza) [reply src/components/viewComponents/Movie.js]: i accidentally realise that im doing 2 PRs, the lazy loader handles by itself the lazy loading with image loading, im just adding extra code for no reason

**Review by JulianPasquale — APPROVED** [review]

**Review by martoesses — APPROVED** [review]

**Pull request merged**